tests/benchmarks/app_kernel/prj_user.conf

```diff
@@ -1,22 +1,25 @@
 CONFIG_TEST=y
 # all printf, fprintf to stdout go to console
 CONFIG_STDOUT_CONSOLE=y
-CONFIG_MAIN_THREAD_PRIORITY=6
-
-CONFIG_FPU=y
-CONFIG_FPU_SHARING=y
 
 # eliminate timer interrupts during the benchmark
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=1
-CONFIG_TICKLESS_KERNEL=n
+CONFIG_TICKLESS_KERNEL=y
```

Comment:
This is `default y if TICKLESS_CAPABLE`. Might be better to filter by ` CONFIG_TICKLESS_CAPABLE` in `testcase.yaml`.

Reply:
I goofed here. The situation we want is as few interrupts as practical. This translates to a tickless kernel (if available) and the number of clock ticks per second to be as low as possible.

Fixing ...